skdownscale/pointwise_models/core.py

```diff
@@ -115,6 +116,22 @@ def _transform_wrapper(X, models, feature_dim=DEFAULT_FEATURE_DIM, **kwargs):
     return xtrans
 
 
+def _getattr_wrapper(models, keys):
+    dims = list(models.dims)
+    shape = list(models.shape)
+    coords = dict(models.coords)
+
+    out = xr.Dataset()
+    for key in keys:
+        out[key] = xr.DataArray(np.empty(shape), coords=coords, dims=dims)
```

Comment:
probably worth specifying the dtype here (and when creating these arrays on individual models). Float64 seems like the logical choice to me

Reply:
hmm some attributes can be boolean or objects i believe. what's the benefit of specifying dtype here?

Reply:
The empty array will always default to being `float64`. That may be what we want but, as you say, some attributes will be other types. I'm honestly not sure what the best thing to do here is.

Reply:
What if we exposed the target type as a class attribute on the Model? 

Something like:

```python
dtypes = models[0, 0].item().attr_dtypes
```

Reply:
since `map_blocks` requires us to provide a template object, I ended up adding an input called dtype and use it to define the data type here.